reconcile/utils/gql.py

```diff
@@ -198,13 +193,39 @@ def get_resource(path: str) -> dict[str, Any]:
     return get_api().get_resource(path)
 
 
+class PersistentRequestsHTTPTransport(RequestsHTTPTransport):
+    def connect(self):
+        if self.session is None:
+            # Copied over from RequestsHTTPTransport
+            self.session = requests.Session()
+            # we did not implement this in our copy!
+            assert self.retries == 0
```

Comment:
we don't retry here since we have the retry mechanism as a decorator, correct?

Reply:
Correct, hence I did not copy that feature over